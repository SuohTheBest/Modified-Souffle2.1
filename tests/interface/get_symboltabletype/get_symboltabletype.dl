.type first_name <: symbol
.type last_name <: symbol
.type gender <: symbol

.decl people (Surname:first_name, name:last_name, gender:gender, age:number, height:float)

.input people ()
.output people ()
