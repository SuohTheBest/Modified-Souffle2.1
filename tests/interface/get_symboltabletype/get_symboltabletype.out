<s:first_name,s:last_name,s:gender,i:number,f:float>

Kelvin	=> 0
Fong	=> 1
M	=> 2
Tom	=> 3
Chan	=> 4
Jack	=> 5
Armstrong	=> 6
Marry	=> 7
Hunter	=> 8
F	=> 9
